docs/source/ml.rst

```diff
@@ -76,20 +98,106 @@ missing values. If you're making a classification model, the model must have a `
+dataset. If you are passing a custom ETL estimator, you will have to
+ensure that no categorical columns remain after the ``transform``
+method is called on the dataset.
+
+.. _hyperparam-search:
+
+Hyperparameter Tuning
+---------------------
+
+You can tune hyperparamters using one of two methods: grid search or
+hyperband. CivisML will perform grid search if you pass a dictionary
+of hyperparameters to the ``cross_validation_parameters`` parameter, where the keys are
+hyperparameter names, and the values are lists of hyperparameter
+values to grid search over. You can run hyperparameter tuning in parallel by
+setting the ``n_jobs``
+parameter to however many jobs you would like to run in
+parallel. ``n_jobs`` defaults to 4.
+
+`Hyperband <https://arxiv.org/abs/1603.06560>`_
+is an efficient approach to hyperparameter optimization, and
+*recommended over grid search where possible*. CivisML will perform
+hyperband optimization for a pre-defined model  if you pass the string
+``'hyperband'`` to ``cross_validation_parameters``. Hyperband is
+currently only supported for the following models:
+``gradient_boosting_classifier``, ``random_forest_classifier``,
+``extra_trees_classifier``, ``multilayer_perceptron_classifier``,
+``stacking_classifier``, ``gradient_boosting_regressor``,
+``random_forest_regressor``, ``extra_trees_regressor``,
+``multilayer_perceptron_regressor``, and ``stacking_regressor``.
+
+Hyperband cannot be used to tune GLMs. For this reason, preset GLMs do
+not have a hyperband option. Similarly, when
+``cross_validation_parameters='hyperband'`` and the model is
+``stacking_classifier`` or ``stacking_regressor``, only the GBT and
+random forest steps of the stacker are tuned using hyperband.
+Note that if you want to use hyperband with a custom model, you will need to
+wrap your estimator in a
+:class:`civismlext.hyperband.HyperbandSearchCV` estimator yourself.
+
+CivisML runs pre-defined with hyperband using the following
+distributions:
+
++------------------------------------+--------------------+-----------------------------------------------------------------------------+
+| Models                             | Cost               | Hyperband                                                                   |
+|                                    | Parameter          | Distributions                                                               |
++====================================+====================+=============================================================================+
+| | gradient_boosting_classifier     | | ``n_estimators`` | | ``max_depth: randint(low=1, high=5)``                                     |
+| | gradient_boosting_regressor      | | ``min = 100,``   | | ``max_features: [None, 'sqrt', 'log2', 0.5, 0.3, 0.1, 0.05, 0.01]``       |
+| | GBT step in stacking_classifier  | | ``max = 1000``   | | ``learning_rate: truncexpon(b=5, loc=.0003, scale=1./167.)``              |
```

Comment:
@jmemich All three of the distributions in the link you highlighted are covered in the documentation here.